tsconfig.json

```diff
@@ -12,5 +12,5 @@
     "strict": true,
     "target": "ES2017"
   },
-  "exclude": ["./dist/**/*"]
+  "exclude": ["./dist/**/*", "./src/**/*.test-d.ts"]
```

Comment:
Just added these two exclusions. These `tsd` modules are never intended to be compiled directly by TypeScript. That project uses a custom TypeScript fork instead.